src/ol/View.js

```diff
@@ -146,9 +146,8 @@ import Units from './proj/Units.js';
  * the animation.
  * @property {ol.Coordinate|undefined} anchor Optional anchor to remained fixed
  * during a rotation or resolution animation.
- * @property {number|undefined} duration The duration of the animation in milliseconds
- * (defaults to `1000`).
- * @property {undefined|function(number):number} easing The easing function used
+ * @property {number|undefined} [duration=1000] The duration of the animation in milliseconds.
+ * @property {undefined|function(number):number} [easing] The easing function used
```

Comment:
Oh sorry for the inaccurate review. You should have removed the `undefined` in all places where you set the type in square parentheses. The babel-plugin-closure-jsdoc will resolve that to the correct type.

Reply:
see #7979